app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -16,14 +16,32 @@ type Props = {
   publicKey: string,
   history: Object
 }
+type State = {
+  intervalId: number
+}
+
+export default class LoginLedgerNanoS extends Component<Props, State> {
+  state = {
+    intervalId: ''
```

Comment:
Any reason this is on the state? (vs just a `let` outside of the class)

Reply:
@evgenyboxer I read that you need to maintain the interval id in state with react if you want to make sure that the interval is cleared when the component unmounts

Reply:
although I think it might work either way. I believe this is best practice